prod-aws/kafka-shared-msk/billing/data-staged-events.tf

```diff
@@ -54,9 +54,16 @@ module "finance_tx_log_staging_connector" {
   produce_topics = [
     kafka_topic.data_staged_events_finance.name,
   ]
-  #  limit to 150 KB per broker, as the consumer doesn't keep up with the producer and the producer is loading the brokers CPU when not throttled.
-  producer_byte_rate = "153600"
-  cert_common_name   = "billing/finance-tx-log-staging-connector"
+  cert_common_name = "billing/finance-tx-log-staging-connector"
+}
+
+resource "kafka_quota" "finance_tx_log_staging_connector" {
+  entity_name = module.finance_tx_log_staging_connector.user_name
```

Comment:
We'll need additional rules to check that the quota is defined for apps in the team module and that they are unique in a module. Will it be easy to do with this? Or with a separate module where we pass again the cert_common_name ?

Reply:
> We'll need additional rules to check that the quota is defined for apps in the team module and that they are unique in a module. Will it be easy to do with this?

Do we _need_ to enforce on these things? Given the effort required to make our linter plugin work for a single set of variables (and even then, only in a limited sense: we require module name to be a string in a module definition) I'm wary of trying to add more variable handling magic there

Reply:
>Do we need to enforce on these things?
I think we do. Otherwise someone may define a quota for another team.
That's why if we introduce our own module, we can suppress having quotas defined directly as resources.
And if we put the cert_common_name on it, we'll only need to check that there is a tls-app in the current module.

Reply:
After talking face to face we concluded that:
- this is good enough for now, as there aren't many quotas defined, and we don't see many definitions in the future, especially put in place by teams.
- we're gonna create an issue to address this validation for the future, to prevent malicious/copy/paste issues.

Reply:
> After talking face to face we concluded that:
> 
>     * this is good enough for now, as there aren't many quotas defined, and we don't see many definitions in the future, especially put in place by teams.
> 
>     * we're gonna create an issue to address this validation for the future, to prevent malicious/copy/paste issues.

Issue: https://linear.app/utilitywarehouse/issue/DENA-1001/add-module-and-linting-for-topic-quotas